packages/itinerary-body/src/AccessLegBody/mapillary-button.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import { StreetView } from "@styled-icons/fa-solid";
+
+/**
+ * Helper method to generate bounding box from a location. Adding the WINDOW to the coordinate
+ * creates a bounding box of approximately 1 meter around the coordinate, which is likely to
+ * encompass any imagery available.
+ * @param coord     The coordinate to convert to a bounding box
+ * @returns         A bounding box 1 meter around the passed coordinate
+ */
+const generateBoundingBoxFromCoordinate = (coord: [number, number]) => {
+  const WINDOW = 0.000075;
+  const south = coord[0] - WINDOW;
+  const north = coord[0] + WINDOW;
+  const west = coord[1] - WINDOW;
+  const east = coord[1] + WINDOW;
+  return [west, south, east, north];
```

Comment:
The bounding box uses NESW instead of "NE corner", "SE corner", etc?

Reply:
[Documentation](https://www.mapillary.com/developer/api-documentation#entities) says "minx,miny,maxx,maxy" which... could be clearer.